diff -Naur original/grib1/5.table all/grib1/5.table
--- original/grib1/5.table	2021-01-21 13:30:31.000000000 +0100
+++ all/grib1/5.table	2022-09-16 17:24:42.510061745 +0200
@@ -8,6 +8,7 @@
 6 6 Average (reference time - P1 to reference time - P2)
 7 7 Average (reference time - P1 to reference time + P2)
 10 10 P1 occupies octets 19 and 20; product valid at reference time + P1
+13 13 Cosmo nudging analysis product for reference time (P1=0)
 51 51 Climatological Mean Value:
 113 113 Average of N forecasts (or initialized analyses); each product has forecast period of P1 (P1=0 for initialized analyses); products have reference times at intervals of P2, beginning at the given reference time.
 114 114 Accumulation of N forecasts (or initialized analyses); each product has forecast period of P1 (P1=0 for initialized analyses); products have reference times at intervals of P2, beginning at the given reference time.
diff -Naur original/grib1/grid_definition_90.def all/grib1/grid_definition_90.def
--- original/grib1/grid_definition_90.def	2021-01-21 13:30:31.000000000 +0100
+++ all/grib1/grid_definition_90.def	2022-09-16 17:25:29.946294523 +0200
@@ -56,7 +56,7 @@
 #Ce      numbers of points.
 #Ce      (Lambert conformal and Mercator are 42 octets in length,
 #Ce      while Space view is 40 for ECMWF (44 in GRIB specification)
-if ( centre != 98 ) {
+if ( centre != 98 and centre != 200  ) {
   pad padding_grid90_1(6);
 }
 
diff -Naur original/grib1/stepType.def all/grib1/stepType.def
--- original/grib1/stepType.def	2021-01-21 13:30:31.000000000 +0100
+++ all/grib1/stepType.def	2022-09-16 17:25:48.898387350 +0200
@@ -6,6 +6,7 @@
 "instant" = {timeRangeIndicator=0;}
 "instant" = {timeRangeIndicator=10;}
 "instant" = {timeRangeIndicator=1;}
+"instant" = {timeRangeIndicator=13;}
 
 "avg"     = {timeRangeIndicator=3;}
 
diff -Naur original/grib2/section.3.def all/grib2/section.3.def
--- original/grib2/section.3.def	2021-01-21 13:30:31.000000000 +0100
+++ all/grib2/section.3.def	2022-09-16 17:26:20.898543879 +0200
@@ -103,6 +103,7 @@
  "cross_section"         = { gridDefinitionTemplateNumber=1000;PLPresent=0; }
  "Hovmoller"             = { gridDefinitionTemplateNumber=1100;PLPresent=0; }
  "time_section"          = { gridDefinitionTemplateNumber=1200;PLPresent=0; }
+ "UTM"                   = { gridDefinitionTemplateNumber=32768;PLPresent=0; }
  "lambert_lam"             = { gridDefinitionTemplateNumber=33; PLPresent=0;  }
  "mercator_lam"            = { gridDefinitionTemplateNumber=13; PLPresent=0;  }
  "polar_stereographic_lam" = { gridDefinitionTemplateNumber=23; PLPresent=0;  }
